easybuild/easyconfigs/b/biogeme/biogeme-3.2.10-foss-2022a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonBundle'
+
+name = 'biogeme'
+version = '3.2.10'
+
+homepage = 'https://pypi.python.org/pypi/biogeme'
+description = """Biogeme is a open source Python package designed for
+the maximum likelihood estimation of parametric models in general,
+with a special emphasis on discrete choice models."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('tqdm', '4.64.0'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('Unidecode', '1.3.6', {
+        'checksums': ['fed09cf0be8cf415b391642c2a5addfc72194407caee4f98719e40ec2a72b830'],
+    }),
+    (name, version, {
+        'checksums': ['60b865e54f4c5ed22eb4a13dc277b72a2fec13e22d243a6f97d05fe7df984d63'],
+    }),
+]
+
+sanity_pip_check = True
+
```

Comment:
There was a `sanity_check_paths` section in a previous version. 

```
sanity_check_paths = {
    'files': [],
    'dirs': ['lib/python%(pyshortver)s/site-packages/biogeme'],
}
```

Should we put that in here too? Or are we checking the existence of standard python package paths in `site-packages` from the EasyBlock side now?

Reply:
Ah, never mind: that's covered by `sanity_pip_check` nowadays.

Reply:
We only specify a custom `sanity_check_paths` if there's more to check for than just a non-empty `lib/python%(pyshortver)s/site-packages` (since `PythonBundle` checks for that by default).
And of course there's also the default `import` check